package.json

```diff
@@ -28,6 +28,11 @@
     "nop": "exit 0"
   },
   "private": true,
+  "pnpm": {
+    "overrides": {
+      "@bundled-es-modules/cookie>cookie": "0.7.1"
```

Comment:
cookie is used only by dev deps (Storybook), so this override seems okay to me. FWIW, path-to-regexp is used only by msw and Storybook as well.